### PR TITLE
Fix SSR Fallback 

### PR DESCRIPTION
**Pull request by ArthurSampaio**
#### What is the purpose of this pull request?

While the shelf is rendered the amount of items is incompatible with the mobile devices.

#### What problem is this solving?

Identifying the client device and displaying an appropriate amount of items. 
_The best approach would be to make a  carousel ([like this ](https://codepen.io/jh3y/pen/WwVKLN) )in pure css to be displayed while the scripts are being loaded, unfortunately it is a costly activity._

#### How should this be manually tested?
https://arthur--storecomponents.myvtex.com/

#### Screenshots or example usage
**Before**
shelf-problem
**After**
pr-shel-nossr

#### Types of changes
- [x] Bug fix (a non-breaking change which fixes an issue)
- [ ] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/ShelfContent.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ const BREAKPOINT_MOBILE_VIEWPORT = 600
 const SLIDER_CENTER_MOBILE_MODE = true
 const ARROWS_MOBILE_VIEWPORT = false
 const DOTS_MOBILE_VIEWPORT = false
+const DEFAULT_ITEMS_MOBILE = 3
```
Comment (brunojdo) [inline react/ShelfContent.js]: Mobile is 1 default item and desktop is 3

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Fixed
+- Limits quantity of items in mobile devices while onSRR.
```
Comment (gugabribeiro) [inline CHANGELOG.md]: Fix typo `onSSR` :pick:

File: react/ShelfContent.js [hunk 3]
```diff
@@ -84,17 +85,18 @@ class ShelfContent extends Component {
   }
 
   ssrFallback() {
-    const { products, itemsPerPage } = this.props
-    const className = this.getClassByItemsPerPage(itemsPerPage)
+    const { products, itemsPerPage, isMobile } = this.props
+    const items = isMobile ? DEFAULT_ITEMS_MOBILE : itemsPerPage
```
Comment (gugabribeiro) [inline react/ShelfContent.js]: It is not better do rename this const value to numberOfItems or something like that? The term `items` is better used when you are assigning an array of items, don't you think?

File: react/ShelfContent.js [hunk 4]
```diff
@@ -84,17 +85,18 @@ class ShelfContent extends Component {
...
+    const className = this.getClassByItemsPerPage(items)
     return (
       <div className="flex justify-center">
         {products &&
-          products.slice(0, itemsPerPage).map(item => {
+          products.slice(0, items).map((item, index) => {
```
Comment (gugabribeiro) [inline react/ShelfContent.js]: This `return` statement is not needed, you can simply do `[...].map((item, index) => ( <div>...</div> )`

File: CHANGELOG.md (hunk 2 above)
Comment (gugabribeiro) [inline CHANGELOG.md]: Fix typo `onSSR` :pick:

File: react/ShelfContent.js (hunk 3 above)
Comment (gugabribeiro) [inline react/ShelfContent.js]: It's not better do rename this `const` value to `numberOfItems` or something like that? The term items is better used when you are assigning an array of items, don't you think?

File: react/ShelfContent.js (hunk 4 above)
Comment (gugabribeiro) [inline react/ShelfContent.js]: This `return` statement is not needed, you can simply do `[...].map((item, index) => ( <div>...</div> )`

Comment (ArthurSampaio) [reply react/ShelfContent.js]: when I do this the `div` content is not rendered on the ui.

**Pull request merged**